backend/battling/tests/test_view.py

```diff
@@ -0,0 +1,154 @@
+from django.test import Client, TestCase
+
+from model_bakery import baker
+
+from battling.models import PokemonTeam
+from pokemon.models import Pokemon
+
+
+class CreateBattleTest(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.client = Client()
+        self.creator = baker.make("users.User")
+        self.creator.set_password("admin")
+        self.creator.save()
+        self.opponent = baker.make("users.User")
+
+    def test_create_zero_battle(self):
+        self.client.login(username=self.creator.email, password="admin")
+
+        battle_data = {
+            "creator": self.creator,
+            "opponent": self.opponent,
+        }
+
+        response = self.client.get("/battle/list/", battle_data)
```

Comment:
Instead of passing the url `"/battle/list/"` use `reverse(namespace)` to make the tests future proof